ext/standard/basic_functions.c

```diff
@@ -1143,6 +1143,11 @@ ZEND_BEGIN_ARG_INFO(arginfo_fmod, 0)
 	ZEND_ARG_INFO(0, y)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO(arginfo_fdiv, 0)
```

Comment:
This looks like it could add Reflection param and return types, with/without being generated by stubs (some of the above functions already do)

$dividend and $divisor could be used as names to match intdiv.

Reply:
I expect that these functions will be converted to stubs very soon, so I'm leaving it for now.